core/src/main/java/com/datastrato/gravitino/listener/event/CreateTableEvent.java

```diff
@@ -0,0 +1,57 @@
+/*
+ *  Copyright 2024 Datastrato Pvt Ltd.
+ *  This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.listener.event;
+
+import com.datastrato.gravitino.NameIdentifier;
+import com.datastrato.gravitino.annotation.DeveloperApi;
+import com.datastrato.gravitino.listener.info.TableInfo;
+
+/**
+ * Represents an event triggered upon the successful creation of a table. This class extends {@link
+ * TableEvent} to provide detailed information specifically related to the table's creation. This
+ * includes the final table information as it is returned to the user once the creation process has
+ * successfully completed.
+ *
+ * <p>Such an event is instrumental for a variety of purposes including, but not limited to,
+ * auditing activities associated with table creation, monitoring the creation of tables within a
+ * system, and acquiring insights into the final configuration and state of a newly created table.
+ */
+@DeveloperApi
+public class CreateTableEvent extends TableEvent {
```

Comment:
Better add `final` constraint to this class.

Reply:
ok

Reply:
done